nautobot/dcim/tests/test_models.py

```diff
@@ -2861,6 +2868,52 @@ def test_module_infinite_recursion_ancestor(self):
 
         self.assertEqual(context.exception.message, "Creating this instance would cause an infinite loop.")
 
+    def test_render_component_names(self):
+        """Test that creating a Module with components properly renders the {module} and {module.parent} variables."""
+        grandparent_module = Module.objects.create(
+            module_type=self.module_type,
+            location=self.location,
+            status=self.status,
+        )
+        grandparent_module_bay = grandparent_module.module_bays.first()
+        grandparent_module_bay.position = "3"
+        grandparent_module_bay.save()
+        parent_module = Module.objects.create(
+            parent_module_bay=grandparent_module.module_bays.first(),
+            module_type=self.module_type,
+            location=self.location,
+            status=self.status,
+        )
```

Comment:
Parent module bay and location are mutually exclusive but it's only caught by clean because Django and MySQL didn't play nicely when putting a database constraint on that. This is a typo in the test

Reply:
```suggestion
        parent_module = Module.objects.create(
            parent_module_bay=grandparent_module.module_bays.first(),
            module_type=self.module_type,
            status=self.status,
        )
        parent_module.clean()
```